### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by jmthomas**

### CODE REVIEW

File: openc3-redis/Dockerfile [hunk 1]
```diff
@@ -16,28 +15,30 @@ ENV REQUESTS_CA_BUNDLE=/devel/cacert.pem
...
-	useradd -r -g redis -u 1001 redis; \
-  apt update && apt upgrade -y; \
+  apk upgrade; \
   fi
 
 RUN mkdir /config
 COPY redis.conf /config/.
 COPY redis_ephemeral.conf /config/.
 COPY users.acl /config/.
-COPY --chmod=0755 ./docker-entrypoint.sh /usr/local/bin/docker-entrypoint.sh
```
Comment (jmthomas) [inline openc3-redis/Dockerfile:33]: This is already part of the alpine docker image so I removed it (don't think it was being used anyway)

**Review by jmthomas — COMMENTED** [review]
Comments inline

File: openc3-redis/Dockerfile (hunk 1 above)
Comment (ryanmelt) [reply openc3-redis/Dockerfile:33]: I modified the default one for a reason.  Probably needs to be put back.

**Review by ryanmelt — CHANGES_REQUESTED** [review]

**Comment by jmthomas** [comment]
Here's the docker file: https://github.com/docker-library/redis/blob/master/7.2/alpine/docker-entrypoint.sh

Looks like you deleted the section in the middle that allows the container to be started with --user. We're not doing that so what's the difference?

**Comment by ryanmelt** [comment]
The gosu and --user stuff is bad because I do run containers with --user and whenever we use compose to a non-default user it does too.

That change needs to be put back.

**Pull request merged**